### DISCUSSION TITLE
ABAPGIT to GHES migration for custom packages

### DISCUSSION THREAD

**Discussion by Ldri-se** [community/community#136932] [Enterprise]
### Select Topic Area

Question

### Body

We migrated the packages from ABAPGIT to GHES, now for few systems we are not able to see even one package in ABAPGIT.

May I know what the reason behind this is? Please let us know. 

Thanks,
Lokesh B.

**Comment by ebndev** [comment]
Hi @SESA754051, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Enterprise` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction.

In the meantime, please feel encouraged to explore existing discussions in the category and utilize the search in case other users may have run into the same issue!
